cid/common.py

```diff
@@ -692,7 +692,11 @@ def status(self, dashboard_id, **kwargs):
                         logger.info(f'Updating dashboard: {dashboard.id} with Recursive = {recursive}')
                         self._deploy(dashboard_id, recursive=recursive, update=True)
                         logger.info('Rediscover dashboards after update')
-                        self.qs.discover_dashboards()
+                        
+                        refresh_overrides = {
```

Comment:
why not just set the list?

Reply:
you are right, a list would be enough, in the end the original one was a flag so, no need for a dictionary unless there are plans to have more functionality and context for a refresh action.

Reply:
refresh_overrides has been changed to a list now.